src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -318,7 +318,11 @@ private void findUnitsThatCantMove(final Map<Territory, ProPurchaseTerritory> pu
           if (ta != null) {
             production = ta.getProduction();
           }
-          if (territoriesToDefendWithOneUnit.contains(t) && unitValue <= (production + 3)) {
+
+          // Only defend territories that either already have units (avoid abandoning territories)
+          // or where unit value is less than production + 3 (avoid sacrificing expensive units to block)
+          if (territoriesToDefendWithOneUnit.contains(t)
```

Comment:
explicit parens would be helpful. The +3 is very confusing, even with the comment. Assigning it to a constant I think would help, simply to give it a name.

If the territory has a unit, we won't abandon it, but could we be over-defending it? What if there is a fighter plane in the territory only, when we could instead put an infantry there?  Perhaps commenting on how we avoid a blitz attack would also be helpful too.